tests/configlet/util/local_test_run.py

```diff
@@ -0,0 +1,135 @@
+#! /usr/bin/env python
```

Comment:
I recommend removing this file. As it will not be regularly executed and maintained, it will eventually grown out of sync with the test and lost its purpose of debugging.